staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -475,6 +488,74 @@ func (f *fancyResponseWriterDelegator) Hijack() (net.Conn, *bufio.ReadWriter, er
 	return f.ResponseWriter.(http.Hijacker).Hijack()
 }
 
+// Small optimization over Itoa
+func versionToString(v int) string {
```

Comment:
what are you using this for?

Reply:
nothing, remnant of earlier iteration that had removedMajor/removedMinor as ints

Reply:
cleaned up